dali/operators/reader/numpy_reader_gpu_op.cc

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "dali/pipeline/data/views.h"
+#include "dali/kernels/transpose/transpose_gpu.h"
+#include "dali/core/static_switch.h"
+#include "dali/operators/reader/numpy_reader_gpu_op.h"
+
+namespace dali {
+
+void NumpyReaderGPU::Prefetch() {
+  // We actually prepare the next batch
+  TimeRange tr("NumpyReaderGPU::Prefetch #" + to_string(curr_batch_producer_), TimeRange::kRed);
```

Comment:
Probably I'm missing something. Can you say, why this is not `DomainTimeRange`, just for my education.

Reply:
It was added before `DomainTimeRange` was introduced and was missed in the rebase. Fixed.